src/components/Users/PreviewUserCard.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 // import {Link} from 'react-router-dom'
-import { Card, Icon, Image } from "semantic-ui-react";
+import { Card, Icon, Image, Button } from "semantic-ui-react";
```

Comment:
lets start removing everything semantic related that isn't an Icon. that way itll be really easy for us to switch over to no framework when josie builds us some icons.